### DISCUSSION TITLE
Resolution scaling for anything below 1280x768 or 1920x1080

### DISCUSSION THREAD

**Discussion by chemlord** [imfing/hextra#419] [Ideas]
Hi I was wondering if there was a way to fix viewing for resolutions below 1280x768 or anything higher than 1920x1080 for wide and normal?

**Comment by imfing** [comment]
Can you elaborate more on the issue you were facing?

**Comment by chemlord** [comment]
Hi Xin, 
![image](https://github.com/imfing/hextra/assets/168226995/0b30f098-7d45-4a05-bd37-6ca24a8fc0f8)

So if you look on my site https://chemlord.github.io/reactionrepo/docs/oxidation-reactions/dess-martin-oxidation/ I'm running into several formatting issues where it appears messed up in several resolutions. Titles not issued properly. Shortcodes such as details not extending fully. I'm not sure if an update took place but my view of my site has been altered significantly. I can provide more pictures if you would like. Also the tabs section in mobile if a title is too long, it will stretch the borders significantly and cause it to bleed through. Please take a look at it.

**Reply by imfing** [reply]
looks like you need to remove this line: 
https://github.com/chemlord/reactionrepo/blob/62d97ce481fa8b62622d5a0adc325aed74b7f450/content/docs/oxidation-reactions/dess-martin-oxidation/_index.md?plain=1#L13

this `div` doesn't have a closing tag `</div>`

**Reply by chemlord** [reply]
Oh wow, thank you this worked. I placed the div and it worked. I kept the line you told me to remove since I'm manually creating tags using the buttons and wish to have some custom spacing between them and my modified docs date and author part.

However, on mobile whenever I view a page with the tabs shortcode and the titles are too long it causes the page to extend too far out. Is there anyway to mitigate this?

![image](https://github.com/imfing/hextra/assets/168226995/eecb936d-70af-4ad1-959f-5a5be2f98937)

**Reply by imfing** [reply]
it's a bug i think.

**Reply by imfing** [reply]
Created a PR to fix it https://github.com/imfing/hextra/pull/422

**Reply by chemlord** [reply]
Thank you so much. Also I have a dumb question. How do I apply this fix without overriding my previously modified files?

**Reply by imfing** [reply]
I've merged the change. See https://github.com/imfing/hextra/discussions/65#discussioncomment-7050702 for how to apply it

**Reply by chemlord** [reply]
> I've merged the change. See [#65 (reply in thread)](https://github.com/imfing/hextra/discussions/65#discussioncomment-7050702) for how to apply it

Hi I've run into another bug (maybe) for the Tabs, so I put a ## Markdown Title inside the tab shortcode, Now I'm trying to create a link that leads to this directly title in my tab shortcode but it won't lead me there. Instead it leads me to the regular page, you can find the button here: https://chemlord.github.io/reactionrepo/articles/ 

The ## Markdown is in the Oxidation in DMF tab. https://chemlord.github.io/reactionrepo/docs/oxidation-reactions/corey-schmidt-oxidation/#primary-and-secondary-alcohols
